### PR TITLE
Signup frontend

### PR DESCRIPTION
**Pull request by Avedis92**
This is the first prototype of both landing page and sign up form.

### CODE REVIEW

File: os-ui/public/categorieData.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
```
Comment (SevakMart) [inline os-ui/public/categorieData.json:1]: please remove mock data

File: os-ui/src/component/Button/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+interface ButtonProps{
```
Comment (SevakMart) [inline os-ui/src/component/Button/Button.tsx:1]: better to group this kind of common elements into the separate folder, so we know they can be used in different components

Comment (Avedis92) [reply os-ui/src/component/Button/Button.tsx:1]: This is component is susceptible to change when updating the design of landing page and sign up form. When I update them I will remove all interfaces to the types folder where I have put the other interfaces

File: os-server/src/main/resources/db/migration/h2/V1_0_0__init.sql [hunk 3]
```diff
@@ -69,7 +69,7 @@ CREATE TABLE IF NOT EXISTS category
             ON UPDATE CASCADE
```
Comment (SevakMart) [inline os-server/src/main/resources/db/migration/h2/V1_0_0__init.sql:69]: please revert sql changes as well

**Review by SevakMart — APPROVED** [review]

**Review by SevakMart — APPROVED** [review]

**Pull request merged**